src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -946,6 +946,20 @@ public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(Rend
                 var description = Resources.RenderingMemoryOutageDescription;
                 (dynamoModel as DynamoModel).Report3DPreviewOutage(summary, description);
             }
+            catch (InstancingRenderFailureException)
+            {
+                //Notify the user of an issue impacting the background preview but do not disable the background preview.
+                string title = Resources.PartialRenderFailureTitle;
+                string summary = Resources.InstancingRenderFailureSummary;
+                var description = Resources.InstancingRenderFailureDescription;
+
+                const string imageUri = "/DynamoCoreWpf;component/UI/Images/task_dialog_future_file.png";
+                var args = new TaskDialogEventArgs(
+                   new Uri(imageUri, UriKind.Relative),
+                   title, summary, description);
+
+                (dynamoModel as DynamoModel).OnRequestTaskDialog(null, args);
```

Comment:
Let's use the `DynamoMessageBox` UI from the `MessageBoxService` for consistency with all our error and warning message boxes. They come with the right styles, icons etc.

Reply:
I don't think there is a DynamoModel method to request that though correct?

Reply:
You can invoke it using `DynamoMessageBox.Show()` like it's  done [here](https://github.com/DynamoDS/Dynamo/blob/78797d8c4a8d2e3c9273b1125a8e4ffc8a1cda4b/src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs#L728).